src/Symfony/Bridge/Twig/Resources/views/Form/foundation_5_layout.html.twig

```diff
@@ -275,9 +275,17 @@
 {# Rows #}
 
 {% block form_row -%}
-    {%- set widget_attr = {} -%}
+    {%- set describedby = '' -%}
+    {%- if errors|length > 0 -%}
+        {%- set describedby = (describedby~" form-error-"~id) | trim -%}
```

Comment:
At this point, I'm wondering if we could create a function/macro/... something that would avoid repeating the same code again and again.